packages/moonstone/Scrollable/ScrollButtons.js

```diff
@@ -272,10 +272,14 @@ class ScrollButtons extends Component {
 
 	onKeyDownPrev = (ev) => {
 		const
-			{focusableScrollButtons} = this.props,
+			{focusableScrollButtons, vertical} = this.props,
 			{nextButtonDisabled, prevButtonDisabled} = this.state,
 			{keyCode} = ev;
 
+		if (!vertical) {
+			return;
```

Comment:
I don't think we should call `return` here. We will need to determine whether we should call `preventDefault(ev)` like below, which will prevent default browser scroll behavior and event bubbling. I think since we are not changing focus, we would want to prevent bubbling in these cases. 

Also, the browser can naturally scroll unexpectedly when attempting to use the paging controls. We will need to prevent that.

Reply:
> Also, the browser can naturally scroll unexpectedly when attempting to use the paging controls. We will need to prevent that.

I also agree with @germboy. Just I want to leave two things for your information.

The first one is that, at least for Chrome, page up/down keys work only for a vertical scroller naturally. I think our components are relying on it. Maybe we need to check it up later if needed.

The second one is that `stopPropagation` and `preventDefault` will be called in `Scrollable` anyway. But that routine is not for scroll buttons but for a whole area. It looks better to prevent in buttons.

Reply:
Good Point! I missed considering event bubbling.

I called `ev.preventDefault()` instead of `preventDefault(ev)` because the Scroller having several horizontal VirtualLists should scroll when pressing a channel up or a down key when focus is on the scroll button of those VirtualLists. So `ev.stopPropagation()` should not be called to pass the event for the Scroller to scroll.

When pressing a channel up or a down key when focus is on the item of those VirtualLists, the Scroller does not scroll. We need to fix it after creating a new issue.

Reply:
I felt like this might cause problems if we applied this for every key rather than just the Page-Up/Down, but it seems like it works correctly for all the scenarios I tried.

Reply:
Actually, I just tried some more things and it does cause problems (For example, try pressing the Print hotkey).  Can we adjust this to only preventDefault if it's a Page Up/Down?

Reply:
Yes. We should do. I wrapped the code with the condition to check if it's Page Up/Down.